app/src/main/java/blockeq/com/stellarwallet/activities/BaseActivity.kt

```diff
@@ -14,8 +14,10 @@ abstract class BaseActivity : AppCompatActivity() {
         if (WalletApplication.appReturnedFromBackground) {
             WalletApplication.appReturnedFromBackground =  false
 
-            if (!WalletApplication.localStore.encryptedPhrase.isNullOrEmpty()) {
+            if (!WalletApplication.localStore.encryptedPhrase.isNullOrEmpty() && !WalletApplication.localStore.stellarAccountId.isNullOrEmpty()) {
                 launchPINView(PinType.LOGIN, "", "", null, true)
+            } else {
+                WalletApplication.localStore.clearUserData()
```

Comment:
please create a method under Account that wipes the wallet.
Settings > clear should call this method too.
Please remove the keystore alias too.